### PR TITLE
Added a quicker way to build visual studio solution.

### PR DESCRIPTION
**Pull request by benbennett**
The command also works for other cmake generators
but needs tweeks to the --config /--target for respective generator.

### CODE REVIEW

**Comment by benbennett** [comment]
CLA signed
I signed it.

File: cmake/README.md [hunk 1]
```diff
@@ -144,6 +144,7 @@ or
 
      C:\Path\to\protobuf\cmake\build\debug>nmake
 
+
```
Comment (xfxyjwf) [inline cmake/README.md]: Remove this empty line?

File: cmake/README.md [hunk 2]
```diff
@@ -152,6 +153,11 @@ If you prefer to use the IDE:
   * Choose "Debug" or "Release" configuration as desired.
   * From the Build menu, choose "Build Solution".
 
+or to build the solution from command line
+
+     C:\Path\to\protobuf\cmake\ cmake --build solution --config Debug --target INSTALL
```
Comment (xfxyjwf) [inline cmake/README.md]: "cmake\" => "cmake>"

**Comment by benbennett** [comment]
will do the corrections thanks

**Pull request closed without merging**